pyServer/manifests/windows/aks

```diff
@@ -19,8 +19,8 @@ copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Admin.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Operational.evtx
 copy,/Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Operational.evtx
 
-
-
+echo,### Calico Windows logs
+copy,/CalicoWindows/logs/*.log
```

Comment:
How many files are we expecting here? As it's '*'? And what is the expected total size?
Also do we know that this logs can not contain any secrets/password? If that's the case then let's add 'noscan' after it.

Reply:
Good question. Normally it contains below 4 log files and the file log size should be in N MB.
I think that it should not contain any secrets/password but let me double check it.
```
    Directory: C:\CalicoWindows\logs


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----         2/2/2021  10:09 AM              0 calico-felix.err.log
-a----         2/3/2021   1:02 AM         165753 calico-felix.log
-a----         2/2/2021  10:09 AM              0 calico-node.err.log
-a----         2/2/2021  10:09 AM           3865 calico-node.log
```

Reply:
Confirmed. @chintanrp 
1. The log will be rotated once per day or after 10MB.
2. The logs do not contain any secrets/password. 
3. Only above 4 log files in this folder.

Reply:
@AbelHu  - As these logs will not have any secrets, let add ",noscan" for that.
like - 
`copy,/CalicoWindows/logs/*.log,noscan`

Reply:
Done